From a2cc32b96ff77add0d6645daceb7dba3a4c94b34 Mon Sep 17 00:00:00 2001
From: "haolong.zhang" <haolong.zhang@ck-telecom.com>
Date: Fri, 22 Aug 2014 17:41:38 +0800
Subject: [PATCH 3/3] [Feature]
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

  Remove FM Related : Factory Item & FM apk & music FM source
    [id]:N/A
    [TYPE]
       FEATURE
    [Products]
       hicam_baseline
    [Requirement]
       N/A
    [Solution Description]
       N/A
    [Other info]
 要提交的变更：
	修改:         mediatek/config/common/ProjectConfig.mk
	修改:         packages/apps/Music/src/com/android/music/MusicFeatureOption.java

Change-Id: Ib3214a1c31e37931340d1e8151a68e65f00d22c7
Signed-off-by: haolong.zhang <haolong.zhang@ck-telecom.com>
---
 mediatek/config/common/ProjectConfig.mk                  | 16 ++++++++--------
 .../Music/src/com/android/music/MusicFeatureOption.java  |  2 +-
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/mediatek/config/common/ProjectConfig.mk b/mediatek/config/common/ProjectConfig.mk
index 491446a..5356984 100755
--- a/mediatek/config/common/ProjectConfig.mk
+++ b/mediatek/config/common/ProjectConfig.mk
@@ -319,14 +319,14 @@ MTK_FD_SUPPORT=yes
 MTK_FILEMANAGER_APP=yes
 
 
-MTK_FMRADIO_APP=yes
-MTK_FM_RECORDING_SUPPORT=yes
-MTK_FM_RX_AUDIO=FM_ANALOG_INPUT
-MTK_FM_RX_SUPPORT=yes
-MTK_FM_SHORT_ANTENNA_SUPPORT=yes
-MTK_FM_SUPPORT=yes
-MTK_FM_TX_AUDIO=FM_ANALOG_OUTPUT
-MTK_FM_TX_SUPPORT=yes
+#MTK_FMRADIO_APP=yes
+#MTK_FM_RECORDING_SUPPORT=yes
+#MTK_FM_RX_AUDIO=FM_ANALOG_INPUT
+#MTK_FM_RX_SUPPORT=yes
+#MTK_FM_SHORT_ANTENNA_SUPPORT=yes
+#MTK_FM_SUPPORT=yes
+#MTK_FM_TX_AUDIO=FM_ANALOG_OUTPUT
+#MTK_FM_TX_SUPPORT=yes
 
 MTK_FOTA_ENTRY=no
 MTK_FOTA_SUPPORT=no
diff --git a/packages/apps/Music/src/com/android/music/MusicFeatureOption.java b/packages/apps/Music/src/com/android/music/MusicFeatureOption.java
index 2fdff65..773402d 100755
--- a/packages/apps/Music/src/com/android/music/MusicFeatureOption.java
+++ b/packages/apps/Music/src/com/android/music/MusicFeatureOption.java
@@ -56,7 +56,7 @@ import com.mediatek.common.featureoption.FeatureOption;
 public final class MusicFeatureOption {
     static final boolean IS_SUPPORT_DRM = FeatureOption.MTK_DRM_APP;
     //static final boolean IS_SUPPORT_THEMEMANAGER = FeatureOption.MTK_THEMEMANAGER_APP;
-    static final boolean IS_SUPPORT_FM_TX = FeatureOption.MTK_FM_TX_SUPPORT;
+    static final boolean IS_SUPPORT_FM_TX = false;//FeatureOption.MTK_FM_TX_SUPPORT;
     public static final boolean IS_SUPPORT_SHARED_SDCARD = FeatureOption.MTK_SHARED_SDCARD;
     static final boolean IS_HOTKNOT_SUPPORTED = FeatureOption.MTK_HOTKNOT_SUPPORT;
 }
-- 
1.9.1

